src/content/docs/plugin/deep-linking.mdx

```diff
@@ -211,6 +252,74 @@ pub fn run() {
   </TabItem>
 </Tabs>
 
+## Testing
+
+There is some caveats on testing deep links for your application.
```

Comment:
```suggestion
There are some caveats on testing deep links for your application.
```
The `on` sounds wrong too but my english hit its limits here lol

Reply:
when? idk i suck 🤣